src/Assertion/CallbackAssertion.php

```diff
@@ -31,14 +33,9 @@ public function __construct($callback)
     }
 
     /**
-     * Assertion method - must return a boolean.
-     *
-     * Returns the result of the composed callback.
-     *
-     * @param Rbac $rbac
-     * @return bool
+     * {@inheritdoc}
      */
-    public function assert(Rbac $rbac)
+    public function assert(Rbac $rbac, string $permission = null, RoleInterface $role = null): bool
     {
         return (bool) call_user_func($this->callback, $rbac);
```

Comment:
Two things:

-  This should use the `($this->callback)()` form, as any callback can be called that way in the PHP 7 series.
- Should _all_ arguments be  passed to the callback?

Reply:
I changed it using `$this->callback()` and passing all the arguments.